var/spack/repos/builtin/packages/vtk-m/package.py

```diff
@@ -161,6 +162,11 @@ def cmake_args(self):
             else:
                 options.append("-DVTKm_USE_64BIT_IDS:BOOL=OFF")
 
+            # Support for the testing header files
+            if "+testlib" in spec:
+                if spec.satisfies('@1.7.0:'):
```

Comment:
```suggestion
            if "+testlib" in spec and spec.satisfies('@1.7.0:'):
```

It just hurts the eyes two conditionals which can be merged into one :)